### PR TITLE
chat box component added 

### PR DESCRIPTION
**Pull request by dilippoudel**
The component is not rendering inside chat tab, there is isssue.

### CODE REVIEW

**Comment by Tsaikkari** [comment]
Import chat box to pages/JobseekerProfile and pages/CompanyProfile and pass the component as a prop into Tabs instance. Should work then.

**Comment by Wyfy0107** [comment]
Can we have some image here as well?

File: src/index.tsx [hunk 1]
```diff
@@ -6,8 +6,8 @@ import axios from 'axios'
 
 import App from './App'
 import makeStore from './redux/store'
-
-axios.defaults.baseURL = 'https://tindev-dev-deploy.herokuapp.com'
+// 'https://tindev-dev-deploy.herokuapp.com'
+axios.defaults.baseURL = 'http://localhost:3000'
```
Comment (Tsaikkari) [inline src/index.tsx]: Change this back to heroku url.

**Review by Tsaikkari — CHANGES_REQUESTED** [review]

**Review by Tsaikkari — APPROVED** [review]
Good.

**Pull request merged**